contents.json

```diff
@@ -16250,6 +16250,49 @@
         "https://is1-ssl.mzstatic.com/image/thumb/Purple62/v4/de/e0/70/dee07035-182f-b5e1-9e48-38c5660dd760/pr_source.png/316x0w.jpg",
         "https://is2-ssl.mzstatic.com/image/thumb/Purple118/v4/fe/8d/03/fe8d03fd-bdc7-dca5-ddd1-57de854e716e/pr_source.png/316x0w.jpg"
       ]
+    },
+    {
+      "title": "Beam",
+      "category-ids": [
+        "social"
+      ],
+      "description": "Reddit client that's built to browse. It lets you enjoy all of Reddit's content in a unique and beautiful package.",
+      "license": "gpl-2.0",
+      "source": "https://github.com/awkward/beam",
+      "itunes": "https://itunes.apple.com/nl/app/beam-for-reddit/id937987469",
+      "stars": 0,
+      "tags": [
+        "swift",
+        "objc"
+      ],
+      "date_added": "May 6 2018",
+      "suggested_by": "@murin-matus",
+      "screenshots": [
+        "https://is5-ssl.mzstatic.com/image/thumb/Purple19/v4/ab/1c/34/ab1c34ca-9913-ef85-ae35-10c2c7d3b7a6/mzl.gkpfevya.png/316x0w.jpg",
+        "https://is3-ssl.mzstatic.com/image/thumb/Purple71/v4/eb/79/57/eb795798-bba3-21d5-6070-d8f508af1be0/mzl.lbvrfpjf.png/316x0w.jpg",
+        "https://is2-ssl.mzstatic.com/image/thumb/Purple71/v4/13/5d/53/135d536b-5ec2-a567-d72a-c5e6fb12f804/mzl.rzeufpyk.png/316x0w.jpg"
+      ]
+    },
+    {
+      "title": "Adguard",
```

Comment:
@murin-matus There should be only one project per pull-request. Could you add `Adguard` with another pull request?